app/src/main/java/ch/epfl/qedit/model/answer/TestAnswerFormat.java

```diff
@@ -0,0 +1,21 @@
+package ch.epfl.qedit.model.answer;
+
+/** Only for test purposes */
+public class TestAnswerFormat extends AnswerFormat {
```

Comment:
Is it really used in some tests ? I searched for it in the project and I didn't see it being used in any test... maybe I am just blind and in this case you have to blame me -_-.

Reply:
It is only used once: in QuizFragmentsTestUsingDB I create a quiz with two different AnswerFormats in it and then in QuestionFragment, testAnswerFormatDispatch() I check that the right format is shown when switching to the second question. But it was more like a proof of concepts for the dispatch etc. As soon that we have another AnswerFormat with a Format, Model and a Fragment we will certainly delete all the code that is associated with this TestAnswerFormat.